Formula/e/envoy.rb

```diff
@@ -65,7 +78,7 @@ def install
 
     system Formula["bazelisk"].opt_bin/"bazelisk", "build", *args, "//source/exe:envoy-static.stripped"
```

Comment:
i think we could disable Wasm build (which not only make the build faster but also avoid quirky build errors as i see) by passing `--define wasm=none`. That is always true for mac build so i think it should be fine for linux as well.

```suggestion
    system Formula["bazelisk"].opt_bin/"bazelisk", "build", `--define`, `wasm=none`, *args, "//source/exe:envoy-static.stripped"
```

Reply:
but ack the tcmalloc error is more fundamental so i don't think this will solve the current error with this

Reply:
good idea. I forgot this was disabled for mac build.

Reply:
It should be `disabled` rather than `none`. Can push an update in a bit.

Would need https://github.com/v8/v8/commit/c814d861f87539c89fc96aed0c6a41ef76f0ff53 to build older v8 with clang 19.

Reply:
oh yeah you are right it's changed to `disabled` now. cc @codefromthecrypt 

https://github.com/envoyproxy/envoy/blob/6dde5284615e10b2f36211e20b673009cd33b410/bazel/BUILD#L530